charts/connect/docs/local-testing.md

```diff
@@ -0,0 +1,130 @@
+# Local Testing Guide
+
+This guide explains how to run the same end-to-end tests locally that run in GitHub Actions.
+
+## Prerequisites
+
+### Required Tools
+- **Helm** (v3.4.1+)
+- **Kind** (Kubernetes in Docker)
+- **yq** (YAML processor)
+- **ct** (chart-testing CLI)
+
+## Quick Start
+
+### 1. Install Dependencies
+```bash
+# Auto-detect OS and install dependencies
+make install
+```
+
+### 2. Set Up Test Environment
+
+#### Set Environment Variables
+```bash
+# Set credentials as environment variables
+export OP_CONNECT_CREDENTIALS=$(cat ./1password-credentials.json) # your 1password-credentials.json file content
+export OP_CONNECT_TOKEN="your-connect-token"
+export OP_SERVICE_ACCOUNT_TOKEN="your-service-account-token"
+
+# Set required test fixture values
+export OP_VAULT_ID="your-vault-id"
+export OP_ITEM_ID="your-item-id"
+export OP_SECRET_VALUE="your-expected-secret-value"
+```
+
+### 3. Run Tests
+```bash
+# Run the complete test suite
+make test-e2e
+```
+
+## Available Commands
+
+### Installation Commands
+- `make install` - Auto-detect OS and install dependencies
+- `make install-macos` - Install dependencies for macOS (Homebrew)
+- `make install-linux` - Install dependencies for Linux
+- `make install-windows` - Install dependencies for Windows (WSL/Git Bash)
+
+### Main Test Commands
+- `make test-e2e` - Run complete end-to-end tests (setup → test → cleanup)
+
+### Utility Commands
+- `make check-deps` - Check if all dependencies are installed
+- `make show-config` - Show current test configuration
+- `make test-e2e-cleanup` - Clean up test environment
+
+## Adding New Test Scenarios
+To add a new test scenario:
+1. Create a new YAML file in `ci/` directory
+2. Configure the desired chart values
+3. The workflow will automatically run tests for this configuration
+
+## Debugging
+
+### Keep Cluster for Debugging
+```bash
+make test-e2e
+# Cluster stays running for inspection
+kubectl get pods -n onepassword-connect-test
+kubectl logs -n onepassword-connect-test <pod-name>
+make test-e2e-cleanup  # Clean up when done
```

Comment:
`test-e2e-cleanup` doesn't exist, but I think `cleanup-test-e2e` is the one you're referring to.

```suggestion
make cleanup-test-e2e  # Clean up when done
```